website/search/elastic_search.py

```diff
@@ -299,10 +329,10 @@ def load_parent(parent_id):
 
 
 def get_doctype_from_node(node):
```

Comment:
preprints are indexed as preprints, nodes are indexed as nodes. Nodes will no longer be indexed as preprints.